.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+.env
+# Local Netlify folder
+.netlify
```

Comment:
if you just want to use my current gitignore and commit that:

node_modules/
build/

# env variables
.env
# Local Netlify folder
.netlify
src/components/apps/dungeon/sounds/*